yb-voyager/src/dbzm/config.go

```diff
@@ -310,16 +310,9 @@ func (c *Config) String() string {
 			c.RunId,
 			c.ExporterRole)
 		if c.SSLRootCert != "" {
```

Comment:
Let's not use this check. 
In all conditions, we should use the user-provided ssl mode and set it in dbzm config. In the current logic, we would only set it, if root-cert was not nil. This is a bit dangerous although it might not cause any issues at the moment. 

Instead, we can add a validation (in the exportDataFromTarget init) that if ssl mode = [require, verify, verify-ca, verify-full], ssl-root-cert should also be provided. 

Here, while preparing the dbzm config, 
- blindly set the ssl mode
- if rootcert is not nil, set rootcert

Reply:
Done